### PR TITLE
typo fixes admin.php

### PR DESCRIPTION
**Pull request by chemezov**
### QA

| Q             | A
| ------------- | ---
| Is bugfix?    | yes/no
| New feature?  | yes/no
| Breaks BC?    | yes/no
| Tests pass?   | yes/no
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: src/messages/ru/admin.php [hunk 1]
```diff
@@ -222,7 +222,7 @@
     'js_ngrest_toggler_success' => '%field% успешно обновлено.',
     'layout_image_filter_selection' => 'Выбирите фильтр для изображения:',
```
Comment (rinat-lucky) [inline src/messages/ru/admin.php:223]: Выбирите -> Выберите

Comment (nadar) [reply src/messages/ru/admin.php:223]: So this would be another typo?

Comment (nadar) [reply src/messages/ru/admin.php:223]: @Antikon maybe, something you can help me?

Comment (rinat-lucky) [reply src/messages/ru/admin.php:223]: > So this would be another typo?

yes

**Review by Antikon — APPROVED** [review]

File: src/messages/ru/admin.php (hunk 1 above)
Comment (nadar) [reply src/messages/ru/admin.php:223]: would you like to send a PR @rinat-lucky ?

Comment (rinat-lucky) [reply src/messages/ru/admin.php:223]: > would you like to send a PR @rinat-lucky ?

little bit later

Comment (nadar) [reply src/messages/ru/admin.php:223]: i think @Antikon already did that for you :-)

**Pull request merged**